collection-scripts/gather_sos

```diff
@@ -8,7 +8,7 @@
 #   SOS_ONLY_PLUGINS: list of sos report plugins to use. Empty string to run
 #                     them all. Defaults to: block,cifs,crio,devicemapper,
 #                     devices,iscsi,lvm2,memory,multipath,nfs,nis,nvme,podman,
-#                     process,processor,selinux,scsi,udev
+#                     process,processor,selinux,scsi,udev,openstack_edpm
```

Comment:
/hold sorry, I don't think I know how to check if this won't break today versions of must gather and sos reports that openstack-k8s-operators and this repo uses in CI

Reply:
lets add the logs plugin to also have journal messages collected

Reply:
we have tested this in the linked ci-framework tooling, it works.
the docs follup is https://github.com/openstack-k8s-operators/openstack-must-gather/pull/18

we can merge this